versions/v1/units.json

```
@@ -7237,6 +7400,23 @@
     "source": "qudt.org",
     "sourceReference": "https://qudt.org/vocab/unit/MIN"
   },
+  {
+    "externalId": "time:mo",
+    "name": "MO",
+    "quantity": "Time",
+    "longName": "Month",
+    "aliasNames": [
+      "month",
+      "mo"
+    ],
+    "symbol": "mo",
+    "conversion": {
+      "multiplier": 2.6298e+6,
+      "offset": 0.0
+    },
+    "source": "UNECE: Codes for Units of Measure Used in International Trade (latest revision) - https://unece.org/trade/uncefact/cl-recommendations",
```

```suggestion
    "source": "UNECE: Codes for Units of Measure Used in International Trade (revision 2021) - https://unece.org/trade/uncefact/cl-recommendations",
```
The current latest revision is from 2021, but they could add a new revision some time in the future, so it's better to pinpoint which one we took as reference.